activity_calendar/migrations/0002_auto_20200814_1431.py

```diff
@@ -1,88 +0,0 @@
-# Generated by Django 2.2.3 on 2020-08-14 12:31
```

Comment:
Can we safely delete migrations?

Reply:
This one was only introduced in this branch, so in this case, yes!

Reply:
I disagree. https://github.com/EricTRL/Squire/blob/master/activity_calendar/migrations/0002_auto_20200814_1431.py

However that commit has not yet been deployed so we're technically in the clear...

Reply:
Woops I forgot this was a new branch!